projects/Cryptpad/demo.nix

```diff
@@ -7,8 +7,8 @@
     settings = {
       httpPort = 9000;
       httpAddress = "0.0.0.0";
-      httpUnsafeOrigin = "http://localhost:${toString config.services.cryptpad.settings.httpPort}";
-      httpSafeOrigin = "http://localhost:${toString config.services.cryptpad.settings.httpPort}";
+      httpUnsafeOrigin = "http://localhost:19000";
+      httpSafeOrigin = "http://localhost:19000";
```

Comment:
I feel like this works around the problem rather than solve it and might also be confusing to the user. For example, why are we setting `httpPort = 9000;` but accessing the service from port `19000`?

What I see as a solution is to have a 1:1 port mapping between the host and guest, but to change all ports in the config examples to be `10000 + current port`. As a user, this makes more sense to me and I wouldn't be confused when I change the config's port as I will get what I expect.

Reply:
I don't think it is a workaround. It is correct configuration for specific piece of software for our specific deployment.
We can add a comment to explain the reason for different port number values.

I don't think we are going to deal with this problem very often with other NGI software.

Reply:
> What I see as a solution is to have a 1:1 port mapping between the host and guest, but to change all ports in the config examples to be 10000 + current port. As a user, this makes more sense to me and I wouldn't be confused when I change the config's port as I will get what I expect.

Yes, this is a good alternative.